dev/source/docs/using-sitl-for-ardupilot-testing.rst

```diff
@@ -797,11 +797,11 @@ Multi-Aircraft SITL with MAVProxy
 
 Multiple aircraft can be simulated in SITL with MAVProxy. This feature owes much to prior work here: https://ardupilot.org/mavproxy/docs/getting_started/multi.html 
 
-To access this feature use both the number (-n) or count (--count) as well as the auto-sysid (--auto-syid) features of sim_vehicle.py like so:
+To access this feature use both the number (-n) or count (--count) as well as the auto-sysid (--auto-syid) and multi-cast features of sim_vehicle.py like so:
```

Comment:
auto-sysid

Reply:
@khancyr txs for reviewing.  I think "auto-sysid" is already mentioned so I think this is OK.

Reply:
ha , it is just that `--auto-syid` is written not `--auto-sysid` : miss a s !

Reply:
oh!  existing problem but fixed, txs!